reconcile/saas_auto_promotions_manager/utils/saas_files_inventory.py

```diff
@@ -30,6 +30,7 @@ def __init__(self, saas_files: Iterable[SaasFile]):
         self._channels_by_name: dict[str, Channel] = {}
         self.subscribers: list[Subscriber] = []
         self.publishers: list[Publisher] = []
+        self._assemble_channels()
```

Comment:
@fishi0x01, adding the initialization of all channels; without that, only channels with active subscribers are honored. Am I right?